object_follower_node:
  hsv_lowerb: [40, 96, 88]
  hsv_upperb: [105, 255, 255]